packages/office-ui-fabric-react/src/components/DatePicker/DatePicker.tsx

```diff
@@ -164,8 +164,11 @@ export class DatePicker extends BaseComponent<IDatePickerProps, IDatePickerState
     let { isDatePickerShown, formattedDate, selectedDate, errorMessage } = this.state;
 
     return (
+      // role='application' is used to work around undesirable behavior in JAWS.
+      // Without role='application', when pressing enter to open the date picker popup,
+      // JAWS swallows the first enter key press and nothing happens.
       <div className={ css('ms-DatePicker', styles.root) } ref='root'>
-        <div ref={ (c): HTMLElement => this._datepicker = c }>
+        <div ref={ (c): HTMLElement => this._datepicker = c } role='application'>
```

Comment:
How are you getting to the textbox with JAWS? I'm not seeing why we need role=application. My guess is you're not configured to auto switch JAWS modes. Does narrator behave the same? If no, I'm inclined to punt as this looks like an AT configurable setting.